core/auditlog/auditlog.go

```diff
@@ -245,13 +248,37 @@ func idString(id uint64) string {
 // primeLogFile ensures the logsink log file is created with the
 // correct mode and ownership.
 func primeLogFile(path string) error {
-	f, err := os.OpenFile(path, os.O_CREATE|os.O_WRONLY, 0600)
+	permissions := os.FileMode(0640)
```

Comment:
Perhaps we have a package level variable in `core/paths` called `LogfilePermission`?
```go
var LogfilePermission = os.FileMode(0640)
```
Then we refer to that when dealing with files rather than scattering the magic constant around.